test/cpp/qps/qps_benchmark_script.bzl

```diff
@@ -56,6 +56,15 @@ def qps_json_driver_batch():
 
 def json_run_localhost_batch():
     for scenario in JSON_RUN_LOCALHOST_SCENARIOS:
+        extra_tags = []
```

Comment:
if we are in a hurry, this change is probably fine, but ideally I'd like to see something less adhoc - we keep doing one-off hacks like this and we end up with messy test configuration where no one knows what gets run with which configuration and why.

It would be good to only have a single point of configuration (what are the configs being run and what are their run restrictions), here, from `qps_benchmark_script.bzl` one has no insight into what will be the "600 or more channels" scenarios and what other important params they have - and this disconnect makes things very confusing and opaque.

Reply:
Been looking at this, opened https://github.com/grpc/grpc/pull/17575 as another alternative - think #17575 might actually be simplest since no need to build system hacks